### PR TITLE
issue=#930 update ReSchedule interface of leveldb thread_pool

### PR DESCRIPTION
**Pull request by 00k**

### CODE REVIEW

File: src/leveldb/include/leveldb/env.h [hunk 1]
```diff
@@ -157,7 +157,7 @@ class Env {
       int64_t wait_time_millisec = 0) = 0;
 
   // Update background task priority with the schedule id return by Schedule
-  virtual void ReSchedule(int64_t id, double prio, int64_t millisec = 0) = 0;
+  virtual void ReSchedule(int64_t id, double prio, int64_t millisec = -1) = 0;
```
Comment (lylei) [inline src/leveldb/include/leveldb/env.h]: 能否加个注释说明一下参数的含义

**Comment by 00k** [comment]
build

**Comment by 00k** [comment]
build

**Comment by 00k** [comment]
build

**Comment by 00k** [comment]
build

**Comment by taocp** [comment]
LGTM

**Comment by taocp** [comment]
LGTM

**Pull request merged**